cel/env_test.go

```diff
@@ -515,6 +517,15 @@ func BenchmarkEnvExtendEagerDecls(b *testing.B) {
 	}
 }
 
+func mustContextProto(t *testing.T, pb proto.Message) Activation {
```

Comment:
nit: unused

Reply:
Looks like all of my FromConfig tests were pruned during a rebase. I've restored them and this function is used now. Thanks!